src/TypeSpec.Extension/Emitter.Csharp/src/lib/operation.ts

```diff
@@ -163,15 +163,16 @@ export function loadOperation(
     for (const res of operation.responses) {
         const body = res.responses[0]?.body;
         if (body?.type) {
+            const bodyType = getEffectiveSchemaType(sdkContext, body.type);
```

Comment:
Just curious, why `getEffectiveSchemaType` will resolve this issue?

Reply:
customer appends some header properties to the body, which makes the body model itself become an anonymous model and lose all the decorators related to paging.